File: 3-Bimestre/C++/ex-6.cpp
#include <iostream>
using namespace std;

int main() {
    float nota, soma = 0, media;
    int contador = 0;

    cout << "Digite uma nota de 0 a 10: ";
    cin >> nota;

    while (nota >= 0 && nota <= 10) {
        soma = soma + nota;
        contador++;
        cout << "Digite outra nota ou um numero fora de 0 a 10 para parar: ";
        cin >> nota;
    }

    if (contador > 0) {
        media = soma / contador;
        cout << "Media: " << media;
    } else {
        cout << "Nenhuma nota valida";
    }

    return 0;
}

File: 3-Bimestre/Turma-A/ex-6.py
soma = 0
contador = 0

while True:
    nota = float(input('Digite uma nota de 0 a 10. Para terminar, digite outro valor: '))
    if 0 <= nota <= 10:
        soma += nota
        contador += 1
    else:
        break

if contador > 0:
    media = soma / contador
    print(f'A média das {contador} notas é: {media:.2f}')
else:
    print('Nenhuma nota válida foi digitada.')
